lms/envs/devstack.py

```diff
@@ -245,6 +245,15 @@ def should_show_debug_toolbar(request):  # lint-amnesty, pylint: disable=missing
 ############## Exams CONFIGURATION SETTINGS ####################
 EXAMS_SERVICE_URL = 'http://localhost:8740/api/v1'
 
+TOKEN_SIGNING.update({
+    'JWT_PUBLIC_SIGNING_JWK_SET': (
+        '{"keys": [{"kid": "token-test-sign", "e": "AQAB", "kty": "RSA", "n": "o5cn3ljSRi6FaDEKTn0PS-oL9EFyv1pI7dRgffQLD1q'
```

Comment:
note: this key should be more specific and will change it's just linked to https://github.com/edx/edx-exams/pull/61 right now